doc/source/cluster/running-applications/job-submission/quickstart.rst

```diff
@@ -104,7 +105,9 @@ To submit the Ray Job, we use ``ray job submit``:
     # Job 'raysubmit_inB2ViQuE29aZRJ5' succeeded
     # ------------------------------------------
 
-By default, this command will run the script on the Ray Cluster and wait until the job has finished. Also note the ``hello world`` printout: the stdout of the job is streamed back to the client.
+This command will run the script on the Ray Cluster and wait until the job has finished. Note that it also streams the stdout of the job back to the client (``hello world`` in this case). Ray will also make the contents of the directory passed as `--working-dir` available to the Ray job by downloading the directory to all nodes in your cluster.
+
+Ray will automatically make the contents of the working directory available to the Ray job. This enables you to focus on iterating on your Ray app instead of worrying about packaging.
```

Comment:
@stephanie-wang looks a little repetitive with your edit. Want me to remove this second paragraph?

Reply:
Ah thanks, meant to remove this part.